spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -43,6 +43,19 @@
 
     @provider_ans2 = ManageIQ::Providers::AnsibleTower::Provider.create(:name => "ansibletest2", :url => "10.8.96.109", :zone => @zone)
     @config_ans2 = ManageIQ::Providers::AnsibleTower::ConfigurationManager.find_by_provider_id(@provider_ans2.id)
+
+    @inventory_group = ManageIQ::Providers::ConfigurationManager::InventoryGroup.create(:name => "testinvgroup", :ems_id => @config_ans.id)
+    @inventory_group2 = ManageIQ::Providers::ConfigurationManager::InventoryGroup.create(:name => "testinvgroup2", :ems_id => @config_ans2.id)
+    @ans_configured_system = ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfiguredSystem.create(:hostname                 => "ans_test_configured_system",
```

Comment:
We have a `:configured_system` factory that can simplify building these records for you.  You can create a `:configured_system_ansible_tower` that inherits from it to get the correct type.